content/SCALE/SCALECLIReference/Directory Service/_index.md

```diff
@@ -3,9 +3,93 @@ title: "Directory Service"
 geekdocCollapseSection: true
 description: "Introduces the TrueNAS CLI directory_service namespace, used to access child namespaces and commands including activedirectory, idmap, kerberos, and ldap." 
 weight: 20
-draft: true
+draft: false
 ---
 
+
 {{< toc >}}
 
-Coming soon!
+
+{{< include file="/_includes/CLIGuideWIP.md" type="page" >}}
+
+{{< include file="/_includes/SCALECLIIntroduction.md" type="page" >}}
+
+## Directory_Services Commands
+
+The **directory_service** namespace has two commands and four child namespaces and is based on functions found in the SCALE API and web UI. 
+It provides access to directory service methods. 
+The four child namespaces have their own commands.
+
+You can enter commands from the main CLI prompt or from the **directory_services** namespace prompt.
+
+### Cache_Refresh Command
+
+The `cache_refresh` command refreshes the directory services cache for users and groups that is used as a backing the user.query and group.query methods. 
```

Comment:
I'm not certain on this. I need to discuss how these other methods work as my tests with these options didn't work. After I work out how these do work then I think I can create snippet text for where these options get mentioned. I'm going to move this PR to draft and try to sort this out this morning/today.

Reply:
I've researched this a bit further and submitting a change. The pending change has relrefs to the Accounts _index.md file. When the PR for the accounts user and groups namespaces is approved and merged these links should change to either the individual namespace articles or it can remain pointing at the index file.